deployment/uberspace-ansible/files/localsettings.j2

```diff
@@ -0,0 +1,17 @@
+ALLOWED_HOSTS = ['{{ url }}']
+DEBUG = False
+
+DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.mysql',
+            'NAME': '{{ ansible_user }}_hagrid',
+            'USER': '{{ ansible_user }}',
+            'PASSWORD': '{{ db_password }}',
+            'HOST': 'localhost',
+            }
+        }
+
```

Comment:
We're not running a debug configuration anymore. While these settings actually require the presence of an https connection (a good idea anyway) and hence make debugging using
`python3 manage.py runserver` locally a pain, there quite important for having a somewhat secure setup in production. Please read the django security best practices for further information.
```suggestion
SECURE_BROWSER_XSS_FILTER = True
SESSION_COOKIE_SECURE = True
CSRF_COOKIE_SECURE =  True
X_FRAME_OPTIONS = 'DENY'
```

Reply:
Great suggestion!